main/src/tools/mdmonitor/gtk-gui/Mono.Instrumentation.Monitor.InstrumentationChartView.cs

```diff
@@ -81,9 +81,9 @@ protected virtual void Build ()
 			this.buttonZoomOut.Relief = ((global::Gtk.ReliefStyle)(2));
 			this.buttonZoomOut.Label = "";
 			global::Gtk.Image w3 = new global::Gtk.Image ();
-			// w3.Pixbuf = global::Stetic.IconLoader.LoadIcon (this, "gtk-zoom-out", global::Gtk.IconSize.Menu);
-			// this.buttonZoomOut.Image = w3;
-			// this.hboxChartBar.Add (this.buttonZoomOut);
+			w3.Pixbuf = global::Stetic.IconLoader.LoadIcon (this, "gtk-zoom-out", global::Gtk.IconSize.Menu);
```

Comment:
how can that be? stetic should be removed

Reply:
@lytico Thanks for the Info. I have temporarly enabled this lines again now, because I analyse which lines can be compiled without errors. I currently have also no idea where this window can be opened from the UI.

But you are right in future we should replace it with something new.

Reply:
that#s interesting that it maybe works - so can be stetic survive, but without the designer???